.github/workflows/a11y.yml

```diff
@@ -0,0 +1,158 @@
+
+      - name: Ensure random values are set
+        run: |
+          echo "Using random values: $PW_BROWSER_VALUE, $PW_WIDTH_VALUE, $PW_HEIGHT_VALUE"
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Install Playwright browsers
+        run: npx playwright install --with-deps
+
+      # This is necessary in order for fetch to work.
+      # Finally, made fetch happen...
+      - name: Build proxy-fetcher dist
+        run: |
+          yarn tsc -b ./packages/proxy-fetcher/tsconfig.json
+
+      - name: Run Playwright tests
+        run: |
+          SEGMENT_INDEX=${{ matrix.shardIndex }} yarn playwright test --project=a11y
+
+      - name: Upload scan report to GitHub Actions Artifacts
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: segment-${{ matrix.shardIndex }}.json
+          path: segment-${{ matrix.shardIndex }}.json
+          retention-days: 7
+
+      - name: Upload failed pages report to GitHub Actions Artifacts
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: failed-pages-segment-${{ matrix.shardIndex }}.json
+          path: failed-pages-segment-${{ matrix.shardIndex }}.json
+          retention-days: 7
+
+  merge-scan-reports:
+    if: always()
+    needs: [playwright-tests]
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Download blob reports from GitHub Actions Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          pattern: segment-*
+          merge-multiple: true
+
+      - name: Merge reports in valid JSON format
+        run: |
+          if [ ! -f segment-*.json ]; then
+              echo "No segment-*.json files found."
+              exit 0
+          fi
+          echo '[' > all-segments.json
+          cat segment-*.json >> all-segments.json
+          sed -i '$s/,$//' all-segments.json
+          echo ']' >> all-segments.json
+          cat all-segments.json
+
+      - name: Upload report
+        uses: actions/upload-artifact@v4
+        with:
+          name: a11y-scan-${{ github.run_attempt }}
+          path: all-segments.json
+          retention-days: 14
+
+  merge-failed-pages-reports:
+    if: always()
+    needs: [playwright-tests]
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Download blob reports from GitHub Actions Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          pattern: failed-pages-segment-*
+          merge-multiple: true
+
+      - name: Merge reports in valid JSON format
+        run: |
+          if [ ! -f failed-pages-segment-*.json ]; then
```

Comment:
the last workflow didn't find any failed pages here, but several links that 502'd were included in the scan upload instead. I'd expect to see pages that 502 in this report instead

```json
  {
    "url": "http://www.va.gov/central-ohio-health-care/work-with-us/internships-and-fellowships/pgy-1-pharmacy-residency-program/",
    "browserName": "webkit",
    "viewportSize": {
      "width": 1920,
      "height": 1440
    },
    "violations": [
      {
        "id": "html-has-lang",
        "impact": "serious",
        "tags": [
          "cat.language",
          "wcag2a",
          "wcag311",
          "TTv5",
          "TT11.a",
          "EN-301-549",
          "EN-9.3.1.1",
          "ACT"
        ],
        "description": "Ensures every HTML document has a lang attribute",
        "help": "<html> element must have a lang attribute",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.8/html-has-lang?application=playwright",
        "nodes": [
          {
            "any": [
              {
                "id": "has-lang",
                "data": {
                  "messageKey": "noLang"
                },
                "relatedNodes": [],
                "impact": "serious",
                "message": "The <html> element does not have a lang attribute"
              }
            ],
            "all": [],
            "none": [],
            "impact": "serious",
            "html": "<html><head><title>502 Bad Gateway</title></head>\n<body>\n<center><h1>502 Bad Gateway</h1></center>\n\n\n</body></html>",
            "target": [
              "html"
            ],
            "failureSummary": "Fix any of the following:\n  The <html> element does not have a lang attribute"
          }
        ]
      }
    ]
  },
```

Reply:
i believe it means it 502'd during the scan

Reply:
yep, I didn't see the 502 in the HTML key till now...I think maybe the HTML could be checked for a string of something like `<title>502 Bad Gateway</title>` but I didn't add that yet. We could do all kinds of checks with the results and modify the report to include/exclude/change output.